manim/mobject/coordinate_systems.py

```diff
@@ -743,13 +751,17 @@ def construct(self):
                     0
                 ],
                 target=x,
-                lower_bound=self.x_range[0],
-                upper_bound=self.x_range[1],
+                lower_bound=0,
+                upper_bound=1,
```

Comment:
**Question :** Why did you change it to 0 and 1 ?

Reply:
We are looking for an alpha value to use in `point_from_proportion`, so it needs to be between 0 and 1, not between `x_range[0]` and `x_range[1]`

Reply:
You're right. Thanks for the clarification.